atlp-rwanda/e-commerce-lydia-32-fn
# Ft-buyerShouldBeAbleToTrackTheirOrderStatus#187570844

### What does this PR do?
This PR aims to implement order tracking functionality for buyers within the application.
### Description of Task to be completed?

- Order History Page:

Displays a list of previous orders with details like order number, total cost, and expected delivery date.
Allows buyers to click on an order to navigate to the order detail page.

- Order Detail Page:

Shows comprehensive information about the selected order, including order status, purchased items, delivery details, and tracking numbers.
Provides a button to easily contact customer support for assistance related to the order.

- Customer Support page

### How should this be manually tested?

- Navigate to the order history page.
- Click on various orders to verify redirection to the correct order detail page.
- On the order detail page, check that all relevant order information is correctly displayed.
- Test the functionality of the customer support button to ensure it directs users to the appropriate support page.
